### ISSUE TITLE
File Manager does not work correct on Xampp (Win)

### ISSUE THREAD

**Issue by Zeromax** [contao/standard-edition#17]
In use: Windows 8.1 with xampp and php 5.5
## Reproduce File upload error:
1. create a Folder in the file Manager
2. save and Close
3. upload a file and select the Folder which we have created
4. Contao Message "A error occured" -> Log: `Details:  Folder "D:\www\standard-edition/files/test" was not mounted or is not a directory` `Function:  Contao\DC_Folder::move`
## Reproduce File upload error in files Folder direct:
1. upload file into files Folder
2. upload files and go back
3. Following Errors:

```
( ! ) Warning: mkdir(): Invalid argument in D:\www\standard-edition\vendor\contao\core-bundle\src\Resources\contao\library\Contao\Files.php on line 87
Call Stack
# Time Memory Function Location
1 0.0003 138904 {main}( ) ..\app.php:0 
2 0.0144 1241592 Symfony\Component\HttpKernel\Kernel->handle( ) ..\app.php:22 
3 0.0599 4148872 Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle( ) ..\bootstrap.php.cache:2429 
4 0.0600 4149544 Symfony\Component\HttpKernel\HttpKernel->handle( ) ..\bootstrap.php.cache:3207 
5 0.0600 4150056 Symfony\Component\HttpKernel\HttpKernel->handleRaw( ) ..\bootstrap.php.cache:3056 
6 0.2443 8153128 call_user_func_array ( ) ..\bootstrap.php.cache:3094 
7 0.2443 8153272 Contao\CoreBundle\Controller\BackendController->mainAction( ) ..\bootstrap.php.cache:3094 
8 0.3448 8953960 Contao\BackendMain->run( ) ..\BackendController.php:51 
9 0.3480 9128104 Contao\Backend->getBackendModule( ) ..\BackendMain.php:130 
10 0.3662 10037904 Contao\DC_Folder->move( ) ..\Backend.php:626 
11 0.3707 10122480 Contao\FileUpload->uploadTo( ) ..\DC_Folder.php:943 
12 0.3782 10213408 Contao\FileUpload->resizeUploadedImage( ) ..\FileUpload.php:164 
13 0.3853 10465960 Contao\Image::resize( ) ..\FileUpload.php:332 
14 0.3853 10466128 Contao\Image::get( ) ..\Image.php:857 
15 0.3972 10776560 Contao\Image->executeResize( ) ..\Image.php:946 
16 0.3979 10778824 Contao\Image->executeResizeGd( ) ..\Image.php:510 
17 0.6311 22894464 Contao\GdImage->saveToFile( ) ..\Image.php:542 
18 0.6325 22958944 Contao\Folder->__construct( ) ..\GdImage.php:179 
19 0.6327 22959912 Contao\Files->mkdir( ) ..\Folder.php:111 
20 0.6327 22960032 mkdir ( ) ..\Files.php:87 

( ! ) Warning: mkdir(): Invalid argument in D:\www\standard-edition\vendor\contao\core-bundle\src\Resources\contao\library\Contao\Files.php on line 87
Call Stack
# Time Memory Function Location
1 0.0003 138904 {main}( ) ..\app.php:0 
2 0.0144 1241592 Symfony\Component\HttpKernel\Kernel->handle( ) ..\app.php:22 
3 0.0599 4148872 Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle( ) ..\bootstrap.php.cache:2429 
4 0.0600 4149544 Symfony\Component\HttpKernel\HttpKernel->handle( ) ..\bootstrap.php.cache:3207 
5 0.0600 4150056 Symfony\Component\HttpKernel\HttpKernel->handleRaw( ) ..\bootstrap.php.cache:3056 
6 0.2443 8153128 call_user_func_array ( ) ..\bootstrap.php.cache:3094 
7 0.2443 8153272 Contao\CoreBundle\Controller\BackendController->mainAction( ) ..\bootstrap.php.cache:3094 
8 0.3448 8953960 Contao\BackendMain->run( ) ..\BackendController.php:51 
9 0.3480 9128104 Contao\Backend->getBackendModule( ) ..\BackendMain.php:130 
10 0.3662 10037904 Contao\DC_Folder->move( ) ..\Backend.php:626 
11 0.3707 10122480 Contao\FileUpload->uploadTo( ) ..\DC_Folder.php:943 
12 0.3782 10213408 Contao\FileUpload->resizeUploadedImage( ) ..\FileUpload.php:164 
13 0.3853 10465960 Contao\Image::resize( ) ..\FileUpload.php:332 
14 0.3853 10466128 Contao\Image::get( ) ..\Image.php:857 
15 0.3972 10776560 Contao\Image->executeResize( ) ..\Image.php:946 
16 0.3979 10778824 Contao\Image->executeResizeGd( ) ..\Image.php:510 
17 0.6311 22894464 Contao\GdImage->saveToFile( ) ..\Image.php:542 
18 0.6325 22958944 Contao\Folder->__construct( ) ..\GdImage.php:179 
19 0.6467 22960008 Contao\Files->mkdir( ) ..\Folder.php:111 
20 0.6468 22960136 mkdir ( ) ..\Files.php:87 

( ! ) Warning: mkdir(): Invalid argument in D:\www\standard-edition\vendor\contao\core-bundle\src\Resources\contao\library\Contao\Files.php on line 87
Call Stack
# Time Memory Function Location
1 0.0003 138904 {main}( ) ..\app.php:0 
2 0.0144 1241592 Symfony\Component\HttpKernel\Kernel->handle( ) ..\app.php:22 
3 0.0599 4148872 Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle( ) ..\bootstrap.php.cache:2429 
4 0.0600 4149544 Symfony\Component\HttpKernel\HttpKernel->handle( ) ..\bootstrap.php.cache:3207 
5 0.0600 4150056 Symfony\Component\HttpKernel\HttpKernel->handleRaw( ) ..\bootstrap.php.cache:3056 
6 0.2443 8153128 call_user_func_array ( ) ..\bootstrap.php.cache:3094 
7 0.2443 8153272 Contao\CoreBundle\Controller\BackendController->mainAction( ) ..\bootstrap.php.cache:3094 
8 0.3448 8953960 Contao\BackendMain->run( ) ..\BackendController.php:51 
9 0.3480 9128104 Contao\Backend->getBackendModule( ) ..\BackendMain.php:130 
10 0.3662 10037904 Contao\DC_Folder->move( ) ..\Backend.php:626 
11 0.3707 10122480 Contao\FileUpload->uploadTo( ) ..\DC_Folder.php:943 
12 0.3782 10213408 Contao\FileUpload->resizeUploadedImage( ) ..\FileUpload.php:164 
13 0.3853 10465960 Contao\Image::resize( ) ..\FileUpload.php:332 
14 0.3853 10466128 Contao\Image::get( ) ..\Image.php:857 
15 0.3972 10776560 Contao\Image->executeResize( ) ..\Image.php:946 
16 0.3979 10778824 Contao\Image->executeResizeGd( ) ..\Image.php:510 
17 0.6311 22894464 Contao\GdImage->saveToFile( ) ..\Image.php:542 
18 0.6325 22958944 Contao\Folder->__construct( ) ..\GdImage.php:179 
19 0.6669 22960016 Contao\Files->mkdir( ) ..\Folder.php:111 
20 0.6735 22960152 mkdir ( ) ..\Files.php:87 

( ! ) Warning: mkdir(): Invalid argument in D:\www\standard-edition\vendor\contao\core-bundle\src\Resources\contao\library\Contao\Files.php on line 87
Call Stack
# Time Memory Function Location
1 0.0003 138904 {main}( ) ..\app.php:0 
2 0.0144 1241592 Symfony\Component\HttpKernel\Kernel->handle( ) ..\app.php:22 
3 0.0599 4148872 Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle( ) ..\bootstrap.php.cache:2429 
4 0.0600 4149544 Symfony\Component\HttpKernel\HttpKernel->handle( ) ..\bootstrap.php.cache:3207 
5 0.0600 4150056 Symfony\Component\HttpKernel\HttpKernel->handleRaw( ) ..\bootstrap.php.cache:3056 
6 0.2443 8153128 call_user_func_array ( ) ..\bootstrap.php.cache:3094 
7 0.2443 8153272 Contao\CoreBundle\Controller\BackendController->mainAction( ) ..\bootstrap.php.cache:3094 
8 0.3448 8953960 Contao\BackendMain->run( ) ..\BackendController.php:51 
9 0.3480 9128104 Contao\Backend->getBackendModule( ) ..\BackendMain.php:130 
10 0.3662 10037904 Contao\DC_Folder->move( ) ..\Backend.php:626 
11 0.3707 10122480 Contao\FileUpload->uploadTo( ) ..\DC_Folder.php:943 
12 0.3782 10213408 Contao\FileUpload->resizeUploadedImage( ) ..\FileUpload.php:164 
13 0.3853 10465960 Contao\Image::resize( ) ..\FileUpload.php:332 
14 0.3853 10466128 Contao\Image::get( ) ..\Image.php:857 
15 0.3972 10776560 Contao\Image->executeResize( ) ..\Image.php:946 
16 0.3979 10778824 Contao\Image->executeResizeGd( ) ..\Image.php:510 
17 0.6311 22894464 Contao\GdImage->saveToFile( ) ..\Image.php:542 
18 0.6325 22958944 Contao\Folder->__construct( ) ..\GdImage.php:179 
19 0.7181 22960016 Contao\Files->mkdir( ) ..\Folder.php:111 
20 0.7182 22960152 mkdir ( ) ..\Files.php:87 

( ! ) Warning: imagejpeg(D:\www\standard-edition/assets/images/2/P1140966_back 4zu3-bdbd4402.jpg): failed to open stream: No such file or directory in D:\www\standard-edition\vendor\contao\core-bundle\src\Resources\contao\library\Contao\GdImage.php on line 193
Call Stack
# Time Memory Function Location
1 0.0003 138904 {main}( ) ..\app.php:0 
2 0.0144 1241592 Symfony\Component\HttpKernel\Kernel->handle( ) ..\app.php:22 
3 0.0599 4148872 Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle( ) ..\bootstrap.php.cache:2429 
4 0.0600 4149544 Symfony\Component\HttpKernel\HttpKernel->handle( ) ..\bootstrap.php.cache:3207 
5 0.0600 4150056 Symfony\Component\HttpKernel\HttpKernel->handleRaw( ) ..\bootstrap.php.cache:3056 
6 0.2443 8153128 call_user_func_array ( ) ..\bootstrap.php.cache:3094 
7 0.2443 8153272 Contao\CoreBundle\Controller\BackendController->mainAction( ) ..\bootstrap.php.cache:3094 
8 0.3448 8953960 Contao\BackendMain->run( ) ..\BackendController.php:51 
9 0.3480 9128104 Contao\Backend->getBackendModule( ) ..\BackendMain.php:130 
10 0.3662 10037904 Contao\DC_Folder->move( ) ..\Backend.php:626 
11 0.3707 10122480 Contao\FileUpload->uploadTo( ) ..\DC_Folder.php:943 
12 0.3782 10213408 Contao\FileUpload->resizeUploadedImage( ) ..\FileUpload.php:164 
13 0.3853 10465960 Contao\Image::resize( ) ..\FileUpload.php:332 
14 0.3853 10466128 Contao\Image::get( ) ..\Image.php:857 
15 0.3972 10776560 Contao\Image->executeResize( ) ..\Image.php:946 
16 0.3979 10778824 Contao\Image->executeResizeGd( ) ..\Image.php:510 
17 0.6311 22894464 Contao\GdImage->saveToFile( ) ..\Image.php:542 
18 0.7546 22958848 imagejpeg ( ) ..\GdImage.php:193 

( ! ) Warning: copy(D:\www\standard-edition/assets/images/2/P1140966_back 4zu3-bdbd4402.jpg): failed to open stream: No such file or directory in D:\www\standard-edition\vendor\contao\core-bundle\src\Resources\contao\library\Contao\Files.php on line 233
Call Stack
# Time Memory Function Location
1 0.0003 138904 {main}( ) ..\app.php:0 
2 0.0144 1241592 Symfony\Component\HttpKernel\Kernel->handle( ) ..\app.php:22 
3 0.0599 4148872 Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle( ) ..\bootstrap.php.cache:2429 
4 0.0600 4149544 Symfony\Component\HttpKernel\HttpKernel->handle( ) ..\bootstrap.php.cache:3207 
5 0.0600 4150056 Symfony\Component\HttpKernel\HttpKernel->handleRaw( ) ..\bootstrap.php.cache:3056 
6 0.2443 8153128 call_user_func_array ( ) ..\bootstrap.php.cache:3094 
7 0.2443 8153272 Contao\CoreBundle\Controller\BackendController->mainAction( ) ..\bootstrap.php.cache:3094 
8 0.3448 8953960 Contao\BackendMain->run( ) ..\BackendController.php:51 
9 0.3480 9128104 Contao\Backend->getBackendModule( ) ..\BackendMain.php:130 
10 0.3662 10037904 Contao\DC_Folder->move( ) ..\Backend.php:626 
11 0.3707 10122480 Contao\FileUpload->uploadTo( ) ..\DC_Folder.php:943 
12 0.3782 10213408 Contao\FileUpload->resizeUploadedImage( ) ..\FileUpload.php:164 
13 0.3853 10465960 Contao\Image::resize( ) ..\FileUpload.php:332 
14 0.3853 10466128 Contao\Image::get( ) ..\Image.php:857 
15 0.3972 10776560 Contao\Image->executeResize( ) ..\Image.php:946 
16 0.8111 10898248 Contao\Files->copy( ) ..\Image.php:516 
17 0.8112 10898488 copy ( ) ..\Files.php:233 
```

4 . Navigate back -> The file is uploaded, now there is the error as described in https://github.com/contao/core/issues/7847

```
( ! ) Warning: filesize(): stat failed for D:\www\standard-edition/D:\www\standard-edition/files/P1140966_back 4zu3.JPG in D:\www\standard-edition\vendor\contao\core-bundle\src\Resources\contao\library\Contao\File.php on line 175
Call Stack
# Time Memory Function Location
1 0.0005 134184 {main}( ) ..\app.php:0 
2 0.0139 1164728 Symfony\Component\HttpKernel\Kernel->handle( ) ..\app.php:22 
3 0.0501 4072048 Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle( ) ..\bootstrap.php.cache:2429 
4 0.0501 4072720 Symfony\Component\HttpKernel\HttpKernel->handle( ) ..\bootstrap.php.cache:3207 
5 0.0501 4073232 Symfony\Component\HttpKernel\HttpKernel->handleRaw( ) ..\bootstrap.php.cache:3056 
6 0.2026 8065880 call_user_func_array ( ) ..\bootstrap.php.cache:3094 
7 0.2026 8066024 Contao\CoreBundle\Controller\BackendController->mainAction( ) ..\bootstrap.php.cache:3094 
8 0.2877 8866664 Contao\BackendMain->run( ) ..\BackendController.php:51 
9 0.2914 9040696 Contao\Backend->getBackendModule( ) ..\BackendMain.php:130 
10 0.3102 9960160 Contao\DC_Folder->showAll( ) ..\Backend.php:626 
11 0.3113 10003528 Contao\DC_Folder->generateTree( ) ..\DC_Folder.php:320 
12 0.3187 10259608 Contao\File->__get( ) ..\DC_Folder.php:2477 
13 0.3187 10259760 filesize ( ) ..\File.php:175 
```

**Comment by Zeromax** [comment]
Interessting: The error Messages from Point 3 are only there if I do not use the debug mode.

**Comment by leofeyer** [comment]
@Zeromax You got things wrong. I have only fixed the issue in Contao 3.5, so please go back to contao/core#7847 and run the tests with the latest `release/3.5.0` branch.

**Comment by Zeromax** [comment]
confirm that with https://github.com/contao/core-bundle/commit/26a0e37ebf26e6714169ec5906ba63d32a6b3b24 everything works fine.
